recipe/meta.yaml

```diff
@@ -9,25 +8,29 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  sha256: cc3a47236ff05da377617ddefa867b7ba983819c664e1afe46249e5b469be464
 
 build:
-  noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  skip: true  # [py37]
```

Comment:
```suggestion
```

Reply:
I would actually remove this because it seems there base is 3.6 [here](https://github.com/r1chardj0n3s/parse/blob/1.19.1/tox.ini#L9)

Reply:
done! Thank you!